packages/pretty-format/src/__tests__/ImmutablePlugins-test.js

```diff
@@ -0,0 +1,319 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+ /* eslint-disable max-len */
+
+'use strict';
+
+const React = require('react');
+const prettyFormat = require('../');
+const Immutable = require('immutable');
+const ImmutableOrderedSet = require('../plugins/ImmutableOrderedSet');
+const ImmutableList = require('../plugins/ImmutableList');
+const ImmutableStack = require('../plugins/ImmutableStack');
+const ImmutableSet = require('../plugins/ImmutableSet');
+const ImmutableMap = require('../plugins/ImmutableMap');
+const ImmutableOrderedMap = require('../plugins/ImmutableOrderedMap');
+
+function assertImmutableObject(actual, expected, opts) {
+  expect(
+    prettyFormat(actual, Object.assign({
+      plugins: [
+        ImmutableOrderedSet, 
+        ImmutableList, 
+        ImmutableOrderedMap,
+        ImmutableMap,
+        ImmutableSet,
+        ImmutableStack,
+      ],
+    }, opts))
+  ).toEqual(expected);
+}
+
+describe('Immutable.OrderedSet plugin', () => {
+  it('supports an empty set', () => {
+    assertImmutableObject(
+      Immutable.OrderedSet([]),
+      'Immutable.OrderedSet []'
+    );
+  });
+
+  it('supports a single string element', () => {
+    assertImmutableObject(
+      Immutable.OrderedSet(['foo']),
+      'Immutable.OrderedSet [ "foo" ]'
+    );
+  });
+
+  it('supports a single integer element', () => {
+    assertImmutableObject(
+      Immutable.OrderedSet([1]),
+      'Immutable.OrderedSet [ 1 ]'
+    );
+  });
+
+  it('supports multiple string elements', () => {
+    assertImmutableObject(
+      Immutable.OrderedSet(['jhon', 'mike', 'cristian']),
+      'Immutable.OrderedSet [ "jhon", "mike", "cristian" ]'
```

Comment:
For `{min: true}` I'd rather print:
```js
Immutable.OrderedSet ["jhon", "mike", "cristian"]
```
and for `{min: false}`:
```js
Immutable.OrderedSet [
  "jhon", 
  "mike", 
  "cristian",
]
```
Just like it works for arrays and objects now.